shader/fine.wgsl

```diff
@@ -138,7 +138,7 @@ fn fill_path(tile: Tile, xy: vec2<f32>) -> array<f32, PIXELS_PER_THREAD> {
     }
     // nonzero winding rule
     for (var i = 0u; i < PIXELS_PER_THREAD; i += 1u) {
-        area[i] = abs(area[i]);
+        area[i] = min(abs(area[i]), 1.0);
```

Comment:
Without comments in the vicinity, I can't easily work out the reason for this change.

I will note that this is a significant change if the value is highly negative; whether that makes sense is a different question.

This does at least agree with the `nonzero` comment

Reply:
This ensures consistent coverage between overlapping and non-overlapping regions in self-intersecting paths. When `alpha < 1` this results in increased brightness in overlapping pixels. Note that this clamp was in the original GLSL code so it’s not technically a change.

Reply:
It's basically the definition of the nonzero fill rule: 0 maps to 0, any other value maps to 1, and this particular formulation gets fractional values for antialiasing basically correct.

Reply:
Oh, my mistake. I didn't see that `abs` wasn't removed